core/src/main/java/org/ehcache/core/EhcacheManager.java

```diff
@@ -463,7 +450,7 @@ public void close() throws Exception {
             throw new RuntimeException(e);
           }
         }
-        this.getLogger().debug("Could not create serializers for " + alias, e);
+        LOGGER.debug("Could not create serializers for " + alias, e);
```

Comment:
This might or might not be serious... what logging do we see now if someone misconfigures a cache that needs serializers?